arb_os/messageBatch.mini

```diff
@@ -121,9 +120,6 @@ public func messageBatch_get(batch: MessageBatch) -> option<(IncomingRequest, Me
             }
         ))
     } else {
-        if (batch.numSoFar > 0) {
-            updateBatchSizeEstimate(batch.numSoFar, batch.template.sender);
```

Comment:
The definition of `updateBatchSizeEstimate` should probably be removed

Reply:
Done.